roadmap.md

```diff
@@ -42,6 +42,7 @@ This scenario loads a web page, enters text into a form field via the keyboard,
 - [x] Everything from the previous scenario
 - [x] [Emulating keyboard input](https://w3c.github.io/webdriver-bidi/#command-input-performActions)
 - [x] [Emulating mouse input](https://w3c.github.io/webdriver-bidi/#command-input-performActions)
+- [ ] [Filling out `<input type=file>`](https://github.com/w3c/webdriver-bidi/issues/494)
```

Comment:
"Filling out" doesn't exactly convey the scope to me. It seems like the proposed scope is "Set local file paths on `<input type=file>`" i.e. this doesn't cover uploading file data or similar.

Reply:
There is now another issue for that with a proposal. As such we can close this one.